iina/MainWindowMenuActions.swift

```diff
@@ -421,4 +421,10 @@ extension MainWindowController {
     }
   }
 
+  @IBAction func menuOpenHistory(_ sender: NSMenuItem) {
+    guard let url = sender.representedObject as? URL else { return }
+    playerCore.playFile(url.path)
+    // FIXME: this line shouldn't be here
```

Comment:
`isNetworkResource` should be set in a more general way; not only when opened from history.